inc/namespace.php

```diff
@@ -22,6 +22,11 @@
 function bootstrap() {
 	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_elasticpress' );
 	add_filter( 'altis_healthchecks', __NAMESPACE__ . '\\add_elasticsearch_healthcheck' );
+
+	// Load debug bar for ElasticPress if Query Monitor is enabled in the config.
+	if ( get_config()['modules']['dev-tools']['query-monitor'] ?? false ) {
+		add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_debug_bar_elasticpress', 0 );
```

Comment:
One last addition here is to add `define( 'WP_EP_DEBUG', true );` to make sure elasticpress queries get logged. Wrap it in a conditional check to make it's not already defined though